app/controllers/admin/edition_images_controller.rb

```diff
@@ -6,7 +6,22 @@ class Admin::EditionImagesController < Admin::BaseController
 
   def index; end
 
+  def confirm_destroy; end
+
+  def destroy
+    image.destroy!
+    redirect_to admin_edition_images_path(@edition), notice: "#{image.image_data.carrierwave_image} has been deleted"
+  end
```

Comment:
My instinct tells me that this is the sort of functionality we should be covering with tests. It's always easier to add tests at the beginning rather than retro-fit them later.

However, I'm struggling to find good examples of other tests in Whitehall which cover the deletion journey. I was hoping there'd be some decent feature specs for attachments, since they follow a similar 'CRUD' workflow. But annoyingly the [edition-attachments.feature](https://github.com/alphagov/whitehall/blob/7ccf59cae75651bd9f295fd305a446c0c3b62f58/features/edition-attachments.feature) spec stops short of testing the deletion journey.

Despite not having a solid example to follow, I still think it'd be good to have the 'D' part of this 'CRUD' journey covered by a feature test. What do you think? 🤔

Reply:
Good point, have added a test now 🌞